ci_tools/prepare_linux.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
We should probably leave the system installation of Postgres alone, on both OS X and Linux. Using a temporary database directory is much more congenial to the development habits of many, who would expect to be able to run tests safely from their laptop.